src/validators/prebuilt.rs

```diff
@@ -0,0 +1,68 @@
+use pyo3::exceptions::PyValueError;
+use pyo3::intern;
+use pyo3::prelude::*;
+use pyo3::types::{PyDict, PyType};
+
+use crate::errors::ValResult;
+use crate::input::Input;
+use crate::tools::SchemaDict;
+
+use super::ValidationState;
+use super::{BuildValidator, CombinedValidator, DefinitionsBuilder, SchemaValidator, Validator};
+
+#[derive(Debug)]
+pub struct PrebuiltValidator {
+    schema_validator: Py<SchemaValidator>,
+    name: String,
+}
+
+impl BuildValidator for PrebuiltValidator {
+    const EXPECTED_TYPE: &'static str = "prebuilt";
```

Comment:
Further thought, it might be better to not have this as a `BuildValidator` implementation and instead just a normal `PrebuiltValidator::try_get_from_schema` method.

The `BuildValidator` implementations are mostly useful for pulling off a corresponding `prebuilt_schema` type, which this currently does not have.

Same for serializers.

Doing this would leave space to add a schema in the future if we so wanted.